config/tuning/m2.yaml

```diff
@@ -0,0 +1,28 @@
+# @package _group_
+search_space:
+  enc_conv_c:
+    - 10
+    - 15
+    - 20
+    - 25
+    - 30
+  enc_kern:
+    - 3
+  enc_hidden:
+    - 64
+    - 128
+    - 192
+    - 256
+    - 320
+  lr:
+    - 1e-5
+    - 1e-1
+  weight_decay:
+    - 1e-6
+    - 1e-2
+max_epochs : 200
+gpus_per_trial: 1
+allocated_gpus: "1, 2, 3, 4"
```

Comment:
Is there any special reason we only want to use these 4 specific gpus for tuning?
@zzhaozhe-profolio

Reply:
no. you can assign any gpus to tune.